src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement, useState } from 'react';
+
+import './App.css';
+import { Counter } from './Counter/Counter.tsx';
+import { Header } from './Header/Header.tsx';
+import { GenreSelect, SearchForm } from './components';
+
+function App(): ReactElement {
+	const [count, setCount] = useState(0);
+	const [searchTerm, setSearchTerm] = useState('');
+	const [selectedGenre, setGenre] = useState<string | null>('All');
+	const [genres] = useState(['All', 'Documentary', 'Comedy', 'Horror', 'Crime']);
+
+	return (
+		<>
+			<Header>
+				<SearchForm searchTerm={searchTerm} onSearch={(setSearchTerm)}></SearchForm>
```

Comment:
I didn't get why we need parentheses around `setSearchTerm`

Reply:
I will undo